### PR TITLE
fix json parsing for reputations as well

### PR DESCRIPTION
**Pull request by aviadl**

### CODE REVIEW

File: content-descriptor.json [hunk 1]
```diff
@@ -1 +1,11 @@
...
\ No newline at end of file
+{
+  "installDate": "0001-01-01T00:00:00Z",
+  "assetId": "REPLACE_THIS_WITH_CI_BUILD_NUM",
+  "releaseNotes": "## Release Notes for version 2.5.0",
```
Comment (meirwah) [inline content-descriptor.json]: shouldn't this be 2.6.1?

File: content-descriptor.json [hunk 2]
```diff
@@ -1 +1,11 @@
...
+  "modified": "REPLACE_THIS_WITH_RELEASE_DATE",
+  "ignoreGit": false,
+  "releaseDate": "REPLACE_THIS_WITH_RELEASE_DATE",
+  "version": -1,
+  "release": "2.5.0",
```
Comment (meirwah) [inline content-descriptor.json]: shouldn't this be 2.6.1?

File: content-descriptor.json (hunk 1 above)
Comment (aviadl) [inline content-descriptor.json]: Doesnt really matter as we create it from flat each build according to the yml file

File: content-descriptor.json (hunk 2 above)
Comment (aviadl) [inline content-descriptor.json]: Doesnt really matter as we create it from flat each build according to the yml file

**Comment by meirwah** [comment]
@aviadl to merge?

**Pull request merged**